includes/class-sensei.php

```diff
@@ -308,6 +317,185 @@ public function __wakeup() {
 		_doing_it_wrong( __FUNCTION__, esc_html__( 'Cheatin&#8217; huh?', 'woothemes-sensei' ), '1.8' );
 	}
 
+	/**
+	 * Show notice when minimum PHP version is not met.
+	 */
+	public static function show_php_notice() {
```

Comment:
I wonder if this function name should be a little less ambiguous? Maybe something like `show_php_version_notice`? Seems like `php_notice` could be interpreted differently.

Reply:
Fixed in c96993192bb3ae9a3f4c665635f2e338f1915263